pkg/crypto/tinkcrypto/primitive/composite/ecdh1pu/ecdh1pu_decrypt_factory.go

```diff
@@ -65,6 +65,7 @@ func newDecryptPrimitiveSet(ps *primitiveset.PrimitiveSet) (*decryptPrimitiveSet
 // Decrypt decrypts the given ciphertext and authenticates it with the given
 // additional authenticated data. It returns the corresponding plaintext if the
 // ciphertext is authenticated.
+// nolint: nestif
```

Comment:
A comment on why nolint is needed would help.

Reply:
Fixed.